app/vep/vep_resources.py

```diff
@@ -23,29 +23,44 @@
 
 from core.logging import InterceptHandler
 from vep.models.upload_vcf_files import Streamer, MaxBodySizeException
+from vep.models.pipeline_model import *
+from vep.utils.nextflow import launch_workflow
+import json
 
 logging.getLogger().handlers = [InterceptHandler()]
 
 router = APIRouter()
 
 
 @router.post("/submissions", name="submit_vep")
-async def submit_vep(request : Request):
+async def submit_vep(request: Request):
     try:
         stream_obj = Streamer(request=request)
```

Comment:
Nitpicking: I would rename, or question, some things here:

- `stream_obj` -> `streamer` (same name as class, but lower-cased), or `request_streamer`
- `stream_result` -> `has_stream_completed`. Although I would drop this variable altogether. Looking at the implementation of the stream method, it can only ever return `True`, or it can error out. But you are already handling your exceptions. What's the point in checking this return value as well?
- `parameters` -> `vep_parameters`, or `vep_job_parameters` (there are so many parameters in this function, ugh)

Reply:
Done